src/buttons/_base.scss

```diff
@@ -159,38 +159,34 @@
 
 [data-button="play"] {
   position: relative;
-  display: inline-block;
-  min-height: 56px;
-  border-radius: 35px;
+  min-height: 3.5rem;
+  border-radius: 2.188rem;
   background-color: color(light-background);
   border: 1px solid color(dark-border);
-  padding: 1.45em size(large) size(normal) size(4.6em);
-  cursor: pointer;
+  padding: ms(0) ms(2) ms(0) 4rem;
+  line-height: 1.4;
   transition: background-color 0.25s;
 
-  @include respond-at-and-above($break-medium) {
-    padding-top: size(normal);
-  }
-
   &:hover {
     background-color: color(medium-background);
   }
 
   @include before() {
-    @include square(40px);
-    left: 8px;
-    top: 7px;
+    @include square(2.5rem);
+    left: 0.5rem;
+    top: 50%;
+    margin-top: -1.25rem;
     border-radius: percentage(1 / 2);
     background: color(background);
     border: 1px solid color(dark-border);
   }
 
   @include after() {
-    @include arrow(right, color(primary), 10px);
-    left: 24px;
-    top: 31px;
-    border-top-width: 6px;
-    border-bottom-width: 6px;
+    @include arrow(right, color(primary), 0.625rem);
+    left: 1.5rem;
+    margin-top: -0.375rem;
+    border-top-width: 0.375rem;
```

Comment:
At the moment all the borders are pixels so +1 for consistency. Feel free to bring up change at the jam though! We want to get input before implementing a change from convention.